lib/src/test/kotlin/graphql/nadel/validation/NadelUnionValidationTest.kt

```diff
@@ -304,7 +304,7 @@ class NadelUnionValidationTest {
                     type Query {
                         entity(id: ID!): Entity
                     }
-                    union Entity = Human | Dog
+                    union Entity = Human | Dog | Issue
```

Comment:
Adding this in to make the fixture more valid.

The test is to check that we run validation on union member types. It was missing `Issue` too, but that's not what we're testing.